testing/integration/groups_test.go

```diff
@@ -30,4 +30,7 @@ const (
 
 	// Upgrade group of tests. Used for testing upgrades.
 	Upgrade = "upgrade"
+
+	// Deb group of tests. Used for testing .deb packages install & upgrades
+	Deb = "deb"
```

Comment:
Why do we define a separate group here? we could run the deb tests on any linux machine so I would be inclined to add them to default/other existing group (we save spawning one extra runner too)

Reply:
> Why do we define a separate group here? we could run the deb tests on any linux machine so I would be inclined to add them to default/other existing group (we save spawning one extra runner too)

Abundance of caution and me not knowing all the internals of the testing suite :-). I'll drop this.  FYI we will have to add Rocky or Alma linux as target distros for the RPM tests.  We really need to test DEB & RPM in their native environments.

Reply:
I mean we don't necessarily need to drop it, putting it into the default group at the moment would save us the overhead. When we add the other runners we may want to re-assess the grouping and see what we can do.
I think ubuntu for the moment should do as a distro, maybe later we want to have proper debian server or other distros that we want to focus our testing on)

Reply:
Please don't drop this. I think the isolation tests between deb/rpm's and `install` command installations are good to have.

Reply:
@pchila & @blakerouse are you OK if we keep the "Deb" group for now, so we can get some deb testing started.  And we come to a consensus when I work on the RPM test cases?

Reply:
Good for me :+1: